### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by Jiack214** [michelelacorte/FlickLauncher#12]
Have you ever considered to submit your app on [F-Droid](https://f-droid.org/)?

Useful links:
Submission Queue: https://gitlab.com/fdroid/rfp/issues
Inclusion Policy: https://f-droid.org/wiki/page/Inclusion_Policy

**Comment by michelelacorte** [comment]
App is on Alpha, I prefer to use Google Play now...

**Comment by IzzySoft** [comment]
@michelelacorte take it the "now" means what it suggests, and you are open to F-Droid later? It's not an "either – or", but an addition. It wouldn't mean much work on your end, but your app would be available to a bigger "audience" (remember there are folks out there who cannot, or do not want to, use Playstore).

Apart from which, being listed on F-Droid means a certain "prestige": due to the strict policies applied there it states your app is privacy friendly, really open source, and "some external source" has checked/verified that. So users haven't to rely on your word alone for this, but see a kind of "proof" :wink:

In short: I'd strongly encourage you considering F-Droid. *You* don't have to publish there, *they* will do that (once [your app has been admitted](https://gitlab.com/fdroid/rfp/issues/317)).
